pyazul/index.py

```diff
@@ -99,8 +99,7 @@ def post_sale_transaction(self, data):
         return self.azul_request(data, operation='ProcessPost')
 
     def verify_transaction(self, data):
-        data.update(validate.verify_transaction(
-            data, operation='VerifyPayment'))
```

Comment:
@crisog validate this with the documentation.

Reply:
Yep, that shouldn't be there, it should be on 'azul_request ' for verify_transaction.